data/styles/clear/include/Icons.mapcss

```diff
@@ -533,9 +549,41 @@ node|z19-[historic=ruins][!name],
 area|z19-[historic=ruins][!name],
 {icon-image: remains-m.svg;}
 
+node|z15-[historic=mine],
+area|z15-[historic=mine],
+{icon-image: mine-m.svg;icon-min-distance: 12;}
+
+node|z16-[historic=aircraft],
+area|z16-[historic=aircraft]
+{icon-image: aircraft-m.svg;icon-min-distance: 12;}
+
+node|z16-[historic=tank],
+area|z16-[historic=tank]
+{icon-image: tank-m.svg;icon-min-distance: 12;}
+
+node|z16-[historic=wreck],
+area|z16-[historic=wreck]
+{icon-image: wreck-m.svg;icon-min-distance: 12;}
+
+node|z16-[historic=locomotive],
+area|z16-[historic=locomotive]
+{icon-image: locomotive-m.svg;icon-min-distance: 12;}
+
+node|z18-[historic=cannon],
+area|z18-[historic=cannon]
+{icon-image: cannon-m.svg;icon-min-distance: 12;}
+
```

Comment:
please remove `icon-min-distance:` from these low importance types, it'll make them less likely to conflict with other icons around

Reply:
Also a nit: when adding new types try to avoid duplicating `node|` & `area|` lines, just a `node|` is enough.
There might be tricky/buggy cases when e.g. removing an `area|` line will break styles due to some peculiar inheritance behavior, but in simple cases like this it should be fine.

Reply:
fixed 👍